src/Aspire.Dashboard/Components/Pages/Resources.razor

```diff
@@ -11,59 +11,62 @@
 
 <div class="content-layout-with-toolbar">
-    <FluentToolbar Orientation="Orientation.Horizontal">
-        <h1 slot="label">@Loc[nameof(Dashboard.Resources.Resources.ResourcesHeader)]</h1>
-        <FluentButton id="typeFilterButton" slot="end"
-                      Appearance="@(AreAllTypesVisible is true ? Appearance.Stealth : Appearance.Accent)"
-                      IconEnd="@(new Icons.Regular.Size24.Filter())"
-                      @onclick="() => _isTypeFilterVisible = !_isTypeFilterVisible"
-                      Title="@(AreAllTypesVisible is true ? Loc[nameof(Dashboard.Resources.Resources.ResourcesTypeFilterAllVisible)] : Loc[nameof(Dashboard.Resources.Resources.ResourcesTypeFiltered)])"
-                      aria-label="@(AreAllTypesVisible is true ? Loc[nameof(Dashboard.Resources.Resources.ResourcesTypeFilterAllVisible)] : Loc[nameof(Dashboard.Resources.Resources.ResourcesTypeFiltered)])" />
-        <FluentPopover AnchorId="typeFilterButton" @bind-Open="_isTypeFilterVisible" AutoFocus="true">
-            <Header>@Loc[nameof(Dashboard.Resources.Resources.ResourcesResourceTypesHeader)]</Header>
-            <Body>
-                <FluentStack Orientation="Orientation.Vertical">
-                    <FluentCheckbox
-                        Label="@ControlsStringsLoc[nameof(ControlsStrings.All)]"
-                        ThreeState="true"
-                        ShowIndeterminate="false"
-                        ThreeStateOrderUncheckToIntermediate="true"
-                        @bind-CheckState="AreAllTypesVisible" />
-                    @foreach (var (resourceType, _) in _allResourceTypes)
-                    {
-                        var isChecked = _visibleResourceTypes.ContainsKey(resourceType);
-                        <FluentCheckbox
-                            Label="@resourceType"
-                            @bind-Value:get="isChecked"
-                            @bind-Value:set="c => OnResourceTypeVisibilityChangedAsync(resourceType, c)"
-                            />
-                    }
-                </FluentStack>
-            </Body>
-        </FluentPopover>
-        <FluentSearch Placeholder="@ControlsStringsLoc[nameof(ControlsStrings.FilterPlaceholder)]"
-                      Immediate="true"
-                      @bind-Value="_filter"
-                      slot="end"
-                      @bind-Value:after="HandleSearchFilterChangedAsync" />
-    </FluentToolbar>
+
     <SummaryDetailsView DetailsTitle="@(SelectedResource != null ? $"{SelectedResource.ResourceType}: {GetResourceName(SelectedResource)}" : null)"
                         ShowDetails="@(SelectedResource is not null)"
                         OnDismiss="@(() => ClearSelectedResourceAsync(causedByUserAction: true))"
                         SelectedValue="@SelectedResource"
                         ViewKey="ResourcesList">
         <Summary>
+
+            <FluentToolbar Orientation="Orientation.Horizontal">
```

Comment:
Doesn't moving the toolbar inside the summary/details view cause it to scroll off screen? If so, I understand the motivation but I think it'd be a regression in the experience for most users. This can be quite a long list, and interacting with the toolbar while scrolling would be common (for toggling masks, searching, etc).

Reply:
Undo this.

As well as the reasons stated above, it changes behavior when the details sidebar is vertical. It's now inconsistent with the rest of the web app.

![image](https://github.com/dotnet/aspire/assets/303201/4623f437-34ab-4f11-9f70-21a63843d758)

Compared to other screens:

![image](https://github.com/dotnet/aspire/assets/303201/044a2b3f-2ad4-4f7c-a5ab-68a72cc760e9)

Reply:
It can be undone, but we cannot keep the behavior as-is. It will not be compliant